tests/e2e/pageObjects/RecipePage.js

```diff
@@ -0,0 +1,74 @@
+const config = require("../../cucumber.conf.js")
+const { format } = require('util')
+
+class RecipePage {
+    constructor() {
+        this.createRecipeDropdown = "//i[@class='fas fa-fw fa-plus fa-lg']"
+        this.createRecipeButton = "//div[@class='dropdown-menu dropdown-menu-center show']/a[@href='/new/recipe/']"
+        this.recipeNameInputSelector = "#id_name"
+        this.saveRecipeNameButtonSelector = "//button[@class='btn btn-success']"
+        this.descriptionInputSelector = "//textarea[@id='id_description']"
+        this.preparationTimeInputSelector = "#id_prep_time"
+        this.waitingTimeInputSelector = "#id_wait_time"
+        this.servingsInputSelector = "#id_servings"
+        this.servingsTextInputSelector = "#id_servings_text"
+        this.imageInputSelector = "//input[@id='id_file_upload']"
+        this.stepNameInputSelector = "//div[@id='id_card_step_0']//div[@class='input-group']/input"
+        this.stepInstructionInputSelector = "//div[@id='id_card_step_0']//textarea"
+        this.timeAddButtonSelector = "//button[contains(text(),'Time')]"
+        this.timeInputSelector = "//label[contains(text(),'Step time in minutes')]/following-sibling::input"
+        this.ingredientsAddButtonSelector = "//button[@class='btn btn-success btn-block']"
+        this.ingredientAmountInputSelector = "//input[@id='amount_0_%s']"
+        this.ingredientUnitInputSelector = "//input[@id='unit_0_%s']"
+        this.ingredientUnitSelectSelector = "//ul[@id='listbox-unit_0_%s']//span[contains(text(),'%s')]//ancestor::li"
+        this.ingredientNameInputSelector = "#ingredient_0_%s"
+        this.ingredientNameSelectSelector = "//ul[@id='listbox-ingredient_0_%s']//span[contains(text(),'%s')]//ancestor::li"
+        this.saveViewOrDeleteButtonSelector = "//button[contains(text(), '%s')]"
+        this.recipeHeadingSelector = "//div[@class='col-12']/h3"
+    }
+
+    async getRecipenName() {
+        return await page.locator(this.recipeHeadingSelector).innerText()
+    }
```

Comment:
async-await can be omitted here. simply return the promise
```suggestion
    getRecipenName() {
        return page.locator(this.recipeHeadingSelector).innerText()
    }
```